torch/csrc/jit/pickler.cpp

```diff
@@ -509,32 +505,33 @@ std::vector<IValue> Unpickler::parse_ivalue_list() {
 
```

Comment:
This comment is meant for line 493. Does the following mean we cannot put multiple `iValue`s onto the stack, instead, we have to nest them into one IValue?

```c++
  TORCH_CHECK(
      stack_.size() == 1,
      "Unpickler expected 1 element on the stack, but found ",
      stack_.size());
```

Reply:
We'd want everything to match Python's `pickle.dump()` which only takes a single value, so everything would have to be wrapped in a tuple by the caller